pkg/iofog/install/remote_agent.go

```diff
@@ -186,6 +186,23 @@ func (agent *RemoteAgent) Stop() (err error) {
 	return
 }
 
+func (agent *RemoteAgent) Prune() (err error) {
+	// Prepare commands
+	cmds := []command{
+		{
+			cmd: "sudo -S service iofog-agent prune",
+			msg: "Pruning Agent " + agent.name,
+		},
+	}
+
+	// Execute commands on remote server
+	if err = agent.run(cmds); err != nil {
```

Comment:
Should only use SSH if detached agent -> Otherwise should use Controller REST API

Reply:
updated